Add first PR template version (#30)
kellerkinderDE/devenv-shopware: .github/pull_request_template.md

<!--
Thank you for contributing to our devenv project!

> It would be helpful, if you gave us as much information as you could to process your pull request better.

Reviewer: ```suggestion
It would be helpful if you could provide us with as much information as possible so we can better process your pull request.
```

Author: Ich würde das `with` noch weglassen wollen. Gibt dem eine etwas andere Bedeutung, finde ich.

Reviewer: To me, it sounds more appropriate with the "with". But both fit well for me

Author: Okay, then let's just take it. ^^

Therefore you are given this description template.
-->

### 1. Why is this change necessary?

### 2. What does this change do, exactly?

### 3. Describe each step to reproduce the issue or behaviour.

### 4. Please link to the relevant issues (if any).

### 5. Checklist

- [ ] I have rebased my changes to remove merge conflicts

> - [ ] I have written or adjusted the documentation according to my changes

Reviewer: ```suggestion
- [ ] I have written or adjusted the README according to my changes
```

Reviewer: due to #33 we have to revert this change 🙈 
Sorry for the inconvenience

Author: It's okay ^^
Reverted the change. 👍
